src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="get" 
+    data-qa="big"
+    >
+      <input 
+        class="input"
+        type="text"
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="get" 
      data-qa="big"
    >
      <input 
        class="input"
        type="text"
```

Reply:
Excuse me, I don't understand how it was appeared, it was incorrect in my file, right?

Reply:
Understand what it means)